src/types/User.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-unused-vars */
```

Comment:
User export changes here - majority of other changes are auto-formatting after adding `.prettierrc`/`.prettierignore` and updating `.eslintrc`.